package.json

```diff
@@ -104,39 +104,39 @@
   },
   "homepage": "https://github.com/uswds/uswds#readme",
   "dependencies": {
-    "classlist-polyfill": "1.0.3",
+    "classlist-polyfill": "1.2.0",
     "object-assign": "4.1.1",
     "receptor": "1.0.0",
     "resolve-id-refs": "0.1.0"
   },
   "devDependencies": {
     "@18f/identity-stylelint-config": "2.0.0",
-    "@babel/core": "7.16.7",
-    "@babel/preset-env": "7.16.7",
-    "@chanzuckerberg/axe-storybook-testing": "6.3.0",
-    "@material-design-icons/svg": "0.11.1",
-    "@storybook/addon-a11y": "6.5.12",
-    "@storybook/addon-essentials": "6.5.12",
-    "@storybook/addon-links": "6.5.12",
-    "@storybook/builder-webpack5": "6.5.12",
-    "@storybook/html": "6.5.12",
-    "@storybook/manager-webpack5": "6.5.12",
-    "@types/node": "16.11.19",
-    "ansi-colors": "4.1.1",
-    "autoprefixer": "10.4.1",
-    "axe-core": "4.6.3",
-    "babel-loader": "8.2.2",
+    "@babel/core": "7.23.2",
+    "@babel/preset-env": "7.23.2",
+    "@chanzuckerberg/axe-storybook-testing": "6.3.1",
+    "@material-design-icons/svg": "0.14.13",
+    "@storybook/addon-a11y": "6.5.16",
+    "@storybook/addon-essentials": "6.5.16",
+    "@storybook/addon-links": "6.5.16",
+    "@storybook/builder-webpack5": "6.5.16",
+    "@storybook/html": "6.5.16",
+    "@storybook/manager-webpack5": "6.5.16",
```

Comment:
Latest 6x StorybookJS version. The next major version, 7 requires major updates. Captured in #5239.